tests_new/integration_tests/core/permissions/test_permissions.py

```diff
@@ -0,0 +1,55 @@
+from assertpy import assert_that
+
+from integration_tests.core.permissions.queries import (
+    update_group_tenant_permissions,
+    list_tenant_permissions,
+    list_tenant_groups,
+)
+from integration_tests.errors import GqlError
+
+
+def test_list_tenant_permissions(clientTenant):
+    response = list_tenant_permissions(clientTenant)
+    assert_that(response).is_not_empty()
+    assert_that(len(response)).is_greater_than_or_equal_to(3)
+    assert_that(response).does_not_contain([None, '', False])
+    assert_that([p.name for p in response]).does_not_contain([None, '', False])
+
+
+def test_list_tenant_permissions_unauthorized(client1):
+    assert_that(list_tenant_permissions).raises(GqlError).when_called_with(client1).contains(
+        'UnauthorizedOperation', 'LIST_TENANT_TEAM_PERMISSIONS'
+    )
+
+
+def test_list_tenant_groups(clientTenant):
+    response = list_tenant_groups(clientTenant)
+    assert_that(response.count).is_greater_than_or_equal_to(4)
+    assert_that(response.nodes).is_not_empty()
+    admin_group = next(group for group in response.nodes if group.groupUri == 'DHAdmins')
+    assert_that(admin_group).contains_key('tenantPermissions')
```

Comment:
nit: any reason we filter for `DHAdmins` and not just take the first entry like `response.nodes[0]` to ensure it contains `tenantPermissions` key?

Reply:
The DHAdmins is a group created during deployment; it is the only one with the tenant permissions. It is true this is testing something else. We could break it down into 2 tests - test_tenant_group_exists and test_list_tenant_groups

Reply:
I ended up adding a comment to explain the purpose of that last check

Reply:
okay I think makes sense - may be no need to include `response.nodes[0]` since it is duplicate chck of `admin_group` which is testing the same and more, but PR looks good